client.go

```diff
@@ -789,7 +794,16 @@ func (c *Client) Route(path string) (err error) {
 	if err != nil {
 		return
 	}
-	return c.dnsProvider.Provide(f)
+	if err = c.dnsProvider.Provide(f); err != nil {
+		return
+	}
+
+	// Return the correct route.
+	instance, err := c.Instances().Remote(ctx, "", path)
+	if err != nil {
+		return
+	}
+	return instance.Route, nil
```

Comment:
@lkingland ok.. this PR includes the change but not who is using this change right? I can see that the route is not used .. yet..